packages/restitution/src/SectionRestitutionColumn.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import {
+  withClassDefault,
+  withClassModifier,
+  WithClassModifierOptions,
+} from '@axa-fr/react-toolkit-core';
+import SectionRestitutionTitle from './SectionRestitutionTitle';
+
+const DEFAULT_CLASSNAME = 'col-sm-12 col-md-12 col-lg-6 col-xl-6';
+
+type SectionRestitutionColumnBaseProps = {
+  className?: string;
+  title?: string;
+  children?: React.ReactNode;
```

Comment:
You maybe can you use native react type PropsWithChildren
```javascript
PropsWithChildren<Partial<{className: string; title: string}>>
```

Reply:
View With Arnaud. We need to update react version for it. I will do it in another PR?